### ISSUE TITLE
Can't access files on https://s3.amazonaws.com/devops-workshop-0526-2051/v1/

### ISSUE THREAD

**Issue by mhauer71** [awslabs/aws-devops-essential#30]
Lab 1, stage 3 asks to download https://s3.amazonaws.com/devops-workshop-0526-2051/v1/Web-App-Archive.zip, but I receive a 403 - Access Denied error. I worked around it using the file https://github.com/awslabs/aws-devops-essential/raw/master/sample-app/Web-App-Archive.zip, but then at the Stage 4 the `aws cloudformation create-stack` fails with
```Error parsing parameter '--template-body': Unable to retrieve https://s3.amazonaws.com/devops-workshop-0526-2051/v1/01-aws-devops-workshop-roles.template: received non 200 status code of 403```.
It seems an issue with the bucket holding the files needed by the workshop.

**Comment by work2fly** [comment]
saw the same issue with the zip. so thanks for that solution.
with the template I got an error when I tried to use the template and also the one I found in the repo but the error message had lots of html so looked like it wasn't reading just the file.
so pointed the command at the git hub raw file.
ie
aws cloudformation create-stack --stack-name DevopsWorkshop-roles \
--template-body https://raw.githubusercontent.com/awslabs/aws-devops-essential/master/templates/01-aws-devops-workshop-roles.template --capabilities CAPABILITY_IAM
and the stack has done something - just need to check it.
so not a solution but a workaround
